stsci-data-analysis/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = 'stsci-data-analysis' %}
-{% set version = '2.0.0' %}
+{% set version = '2.0.1' %}
```

Comment:
Maybe we can leave this as `2.0.0` since we haven't build the overhauled packages yet.